lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -760,7 +766,10 @@ export abstract class CacheManager implements ICacheManager {
      * @param credentialType
      */
     private matchCredentialType(entity: CredentialEntity, credentialType: string): boolean {
-        return (entity.credentialType && credentialType.toLowerCase() === entity.credentialType.toLowerCase());
+        if (entity.credentialType && credentialType.toLowerCase() === entity.credentialType.toLowerCase()) {
+            return true;
+        }
+        return false;
```

Comment:
I don't think this change is needed unless you want to add log messages (probably not needed).

Reply:
Right, made this change to debug, reverting.